suite/memory.x

```diff
@@ -0,0 +1,15 @@
+# memory layout of the FPGA
```

Comment:
Should this be the memory map of the ibex core instead of the FPGA?

Reply:
Since I first developed the suite for a general riscv32imc architecture this is all developed for the qemu emulator. Code specific to the ibex core has not been added before pull request #2.